h/h_api/bulk_api/executor.py

```diff
@@ -0,0 +1,71 @@
+"""Implementations of an 'Executor' responsible for running bulk commands."""
+from h.h_api.bulk_api.model.report import Report
+from h.h_api.enums import CommandResult, CommandType
+
+
+class Executor:
+    """A callback to provide concrete functionality for actions in BulkJob."""
+
+    def configure(self, config):
+        """
+        Configure the job from the provided config object.
+
+        :param config: A config object
+        :return:
```

Comment:
?

Reply:
Looks like a bit of pycharm cruft mixed with some good ole' fashioned Betts vagueness

Reply:
I've made this clearer now. This isn't the entire config object, it's only options deemed relevant to this action for the executor (plenty is handled for you).